README.md

```diff
@@ -39,6 +37,13 @@ Our (work-in-progress) documentation resources include:
  * [Verus License](LICENSE)
  * [Verus Logos](https://verus-lang.github.io/verus/verus/logo.html)
 
+## Examples of Using Verus
+In addition to the documentation above, it can be helpful to see Verus used in action.  Here are some starting points.
+ * <a href="https://verus-lang.github.io/verus/publications-and-projects/">Publications and projects</a> using Verus -- If you're using Verus please consider adding your project to that page (see the instructions there).
+ * [Standalone examples](https://github.com/secure-foundations/human-eval-verus/) showing Verus in use for small, concrete tasks.
+ * [Miscellaneous small examples](source/rust_verify/example) illustrating various Verus features
```

Comment:
These aren't just "small examples". For examples, the `state_machines` directories has sizable examples of VerusSync. Can we make it sound a bit more appealing than "miscellaneous"? I think browsing these would be worthwhile for a newcomer, especially `syntax.rs` (which we may want to point out explicitly), although giving it a bit more structure would be helpful.

Reply:
True, there are some medium-sized examples, but some of them are quite small, e.g., [`adts_eq.rs`](https://github.com/verus-lang/verus/blob/main/source/rust_verify/example/adts_eq.rs), [recommends.rs](https://github.com/verus-lang/verus/blob/main/source/rust_verify/example/recommends.rs), and [debug.rs](https://github.com/verus-lang/verus/blob/main/source/rust_verify/example/debug.rs).  What do you think would be a good way to describe them?

Reply:
"Small and medium-sized examples illustrating ..." ?

Reply:
Sounds good.  Done.